src/LisExpr.chpl

```diff
@@ -37,8 +42,7 @@ module LisExpr
         proc copy(): owned GenListValue throws {
           select (this.lvt) {
             when (LVT.Lst) {
-              var copyList = copyOwnedList(this.toListValue(GenList).lv);
-              return new owned ListValue(copyList);
+              return new owned ListValue(this.toListValue(Symbol).lv);
```

Comment:
is this line correct? shouldn't this be a `GenList` and not a `Symbol`?

Reply:
Oops, you are right, corrected.